Kitodo/src/main/java/org/kitodo/production/renderer/KitodoMediaRenderer.java

```diff
@@ -0,0 +1,134 @@
+
+    @Override
+    public void encodeEnd(FacesContext context, UIComponent component) throws IOException {
+        Media media = (Media) component;
+        ResponseWriter writer = context.getResponseWriter();
+        String src;
+        try {
+            src = getMediaSrc(context, media);
+        } catch (Exception ex) {
+            throw new IOException(ex);
+        }
+
+        Object value = media.getValue();
+        if (value instanceof StreamedContent) {
+            if (PLAYER_HTML_VIDEO.equals(media.getPlayer())) {
+                buildVideoTag(context, media, writer, src);
+                return;
+            } else if (PLAYER_HTML_AUDIO.equals(media.getPlayer())) {
+                buildAudioTag(context, media, writer, src);
+                return;
+            }
+        }
+
+        super.encodeEnd(context, component);
+    }
+
+    private void buildVideoTag(FacesContext context, Media media, ResponseWriter writer, String src)
+            throws IOException {
+        writer.startElement("video", media);
+        Optional<UIComponent> controlsParameter = media.getChildren().stream()
+                .filter(param -> "controls".equals(((UIParameter) param).getName())).findFirst();
+        if (!controlsParameter.isPresent() || !Boolean.FALSE.toString()
+                .equals(((UIParameter) controlsParameter.get()).getValue())) {
+            writer.writeAttribute("controls", "", null);
+        }
+        buildMediaSource(context, media, writer, src);
+        writer.write("Your browser does not support the video tag.");
+        writer.endElement("video");
+    }
+
+    private void buildAudioTag(FacesContext context, Media media, ResponseWriter writer, String src)
+            throws IOException {
+        writer.startElement("audio", media);
+        writer.writeAttribute("controls", "", null);
+        buildMediaSource(context, media, writer, src);
+        writer.write("Your browser does not support the audio tag.");
+        writer.endElement("audio");
+    }
+
+    private void buildMediaSource(FacesContext context, Media media, ResponseWriter writer, String src)
+            throws IOException {
+        if (media.getStyleClass() != null) {
+            writer.writeAttribute("class", media.getStyleClass(), null);
+        }
+
+        renderPassThruAttributes(context, media, HTML.MEDIA_ATTRS);
+        writer.startElement("source", media);
+        writer.writeAttribute("src", src, null);
+
+        Optional<UIComponent> typeParameter = media.getChildren().stream()
+                .filter(param -> "type".equals(((UIParameter) param).getName())).findFirst();
+        if (typeParameter.isPresent()) {
+            writer.writeAttribute("type", ((UIParameter) typeParameter.get()).getValue(), null);
+        }
+
+        writer.endElement("source");
+    }
+
+    private void buildObjectTag(FacesContext context, Media media, ResponseWriter writer, String src, boolean isIE,
```

Comment:
This method seems to be never used. Do we really need it? Otherweise, please add it once it is required.